### ISSUE TITLE
what is Val dataset?

### ISSUE THREAD

**Issue by PatrickStarMS** [hyyan2k/LiSenNet#4]
Hi,I want to konw  what is the val dataset you used?

**Comment by hyyan2k** [comment]
Since VoiceBank+DEMAND datatset only contains training set and test set, we used test set as the val set, following many other studies.

**Comment by cctz111222** [comment]
hi，How does the author handle the val dataset during training? Running on a test set of 872 data, I would run very slowly.

**Comment by hyyan2k** [comment]
Hello, we run validation on the whole 872 data, and the validation time should be far less than training time. If there are no bugs, maybe you could use part of them to validate.
